src/repositories/component/component-graph.repository.ts

```
@@ -0,0 +1,390 @@
+        relationshipMatchClause = `(ctx)-[:CONTEXT_OF]->(item)`;
+        break;
+      case 'Decision':
+        itemMatchClause = `(item:Decision {graph_unique_id: '${escapedItemGraphUniqueId}'})`;
+        relationshipMatchClause = `(ctx)-[:CONTEXT_OF_DECISION]->(item)`;
+        break;
+      case 'Rule':
+        itemMatchClause = `(item:Rule {graph_unique_id: '${escapedItemGraphUniqueId}'})`;
+        relationshipMatchClause = `(ctx)-[:CONTEXT_OF_RULE]->(item)`;
+        break;
+      default:
+        const exhaustiveCheck: never = itemType;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)